todo_app/trello_client.py

```diff
@@ -0,0 +1,72 @@
+from todo_app.item import Item
+import requests
+
+import os
+TRELLO_API_KEY=os.getenv('TRELLO_API_KEY')
+TRELLO_API_TOKEN=os.getenv('TRELLO_API_TOKEN')
+
+def get_cards():
+    url = "https://api.trello.com/1/boards/607d74ceadb03648ed69404d/cards"
+
+    query = {
+    'key': TRELLO_API_KEY,
+    'token': TRELLO_API_TOKEN
+    }
+
+    response = requests.get(
+    url,    
+    params=query
+    ).json()
+    
+    cards=[]
+    for item in response:
+        cards.append(Item(item['idBoard'], item['id'], get_list_name(item['idList']), item['name']))
```

Comment:
This is going to be slow if we have lots of cards, since we'll make a request to Trello for each card to get the list name.

It's not a problem for the exercise, but worth thinking about how you could make it faster if you wanted to.